### ISSUE TITLE
 Add new columns for comments linked to IP or MAC

### ISSUE THREAD

**Issue by guillermonestord** [angryip/ipscan#420]
It would be very useful to be able to add columns linked to the MAC or IP to generate comments.

**Comment by JonnyTech** [comment]
![image](https://github.com/angryip/ipscan/assets/3584732/1c68d259-7948-4c14-a398-73082734efc2)
![image](https://github.com/angryip/ipscan/assets/3584732/6834c4cb-45bb-4085-943e-b495ff14c110)
![image](https://github.com/angryip/ipscan/assets/3584732/a7aeb5a6-f1e7-439e-881b-e11b2b64ba56)
![image](https://github.com/angryip/ipscan/assets/3584732/45b0eb06-7056-486e-9101-4816c8aa02de)

**Comment by guillermonestord** [comment]
Yes, I already saw the possibility of generating 1 comment column, but I would like to add others, where I can change the column label (for example sector, users, etc.) and that can be linked to the MAC or IP.
In this case, comments are linked to the MAC of the device.

And it also has a problem. If close and open the program again, there are comments that do not appear. I have to double click on the line to show them.
